server/plugin/internal/graphql/pullrequest.go

```diff
@@ -0,0 +1,87 @@
+package graphql
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/shurcooL/githubv4"
+
+	"github.com/mattermost/mattermost-plugin-github/server/plugin/internal/graphql/model"
+)
+
+// PullRequestService is responsible for handling pull request related queries
+type PullRequestService service
+
+// Get retrieves pull requests and returns them as []model.PullRequest
+func (p *PullRequestService) Get() ([]model.PullRequest, error) {
+	params := map[string]interface{}{
+		"prSearchQueryArg": githubv4.String(fmt.Sprintf("author:%s is:pr is:%s archived:false", p.client.username, githubv4.PullRequestStateOpen)),
+	}
+	if p.client.org != "" {
+		params["prSearchQueryArg"] = githubv4.String(fmt.Sprintf("org:%s %s", p.client.org, params["prSearchQueryArg"]))
+	}
+
+	var query prSearchQuery
+	var res []model.PullRequest
+
+	if p.test.isTest {
+		query = p.test.clientResponseMock.(prSearchQuery)
+	} else if err := p.client.executeQuery(&query, params); err != nil {
+		return res, err
+	}
+
+	var wg sync.WaitGroup
+
+	for _, resp := range query.Search.Nodes {
+		var requestedReviewers []string
+		var reviews []model.PullRequestReview
+
+		wg.Add(2)
```

Comment:
Spawning two go routing to do a simple loop over an array seems excessive and will most likely not improve performance.